pyproject.toml

```diff
@@ -81,7 +80,6 @@ invoke = "*"
 ipython = "*"
 # Python implementation of markdownlint
 pymarkdownlnt = [
-    {version = "0.9.26", python = "~3.8.0"},
     {version = "~0.9.29", python = ">=3.9,<3.13"},
 ]
```

Comment:
Probably can do a `poetry add --group dev pymarkdownlnt~0.9.29` to fix/simplify this entry?